modules/aws/dynamodb.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/terratest/modules/logger"
+)
+
+func CreateDynamodbTable(t *testing.T, region string, name string) {
+	err := CreateDynamodbTableE(t, region, name)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func CreateDynamodbTableE(t *testing.T, region string, name string) error {
+	logger.Logf(t, "Creating DynamoDb table '%s' in %s", name, region)
+
+	dynamodbClient, err := NewDynamodbClientE(t, region)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Methods that end with `E` should return errors, not call `t.Fatal`

Reply:
This method is called CreateDynamodbTable, and returns Fatal upon encountering an error.
The method CreateDynamodbTableE (which is called by the previous one) returns an error, and does not throw fatal. I think this part is correct, but please reevaluate it.